app/src/screens/StakingTokens.js

```diff
@@ -0,0 +1,218 @@
+          </Field>
+        </div>
+      </div>
+      {myStakes && myStakes.length > 0 && (
+        <div
+          css={`
+            border-bottom: 1px solid ${theme.border};
+          `}
+        >
+          <div
+            css={`
+              padding: ${3 * GU}px;
+            `}
+          >
+            <Field
+              label="Your active tokens"
+              css={`
+                margin-bottom: 0px;
+              `}
+            >
+              <div
+                css={`
+                  ${textStyle('title2')};
+                  color: ${theme.infoSurfaceContent};
+                `}
+              >
+                {`${
+                  myActiveTokens
+                    ? formatTokenAmount(
+                        parseInt(myActiveTokens),
+                        parseInt(stakeToken.tokenDecimals)
+                      )
+                    : '-'
+                } ${stakeToken.tokenSymbol}`}
+              </div>
+              <div
+                css={`
+                  ${textStyle('body4')};
+                  color: ${theme.contentSecondary};
+                `}
+              >
+                Total Active Tokens:{' '}
+                {formatTokenAmount(
+                  parseInt(totalActiveTokens),
+                  parseInt(stakeToken.tokenDecimals)
+                )}{' '}
+                {stakeToken.tokenSymbol}
+              </div>
+            </Field>
+
+            <Field
+              label="SUPPORTED PROPOSALS"
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+            >
+              <div
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+              >
+                <Distribution
+                  heading="Your active token distribution"
+                  items={stakes}
+                  renderLegendItem={({ item }) => {
+                    return (
+                      <div
+                        css={`
+                          background: #daeaef;
+                          border-radius: 3px;
+                          padding: ${0.5 * GU}px ${1 * GU}px;
+                          width: ${compact ? '100%' : `${18 * GU}px`};
+                          text-overflow: ellipsis;
+                          overflow: hidden;
+                          white-space: nowrap;
+                        `}
+                      >
+                        {item}
+                      </div>
+                    )
+                  }}
+                />
+              </div>
+            </Field>
+          </div>
+        </div>
+      )}
+      <div
+        css={`
+          padding-left: ${3 * GU}px;
+          padding-right: ${3 * GU}px;
+          padding-top: ${3 * GU}px;
+        `}
+      >
+        <Field label="Your inactive tokens">
+          <div
+            css={`
+              ${textStyle('title2')};
+            `}
+          >
+            {`${
+              inactiveTokens
```

Comment:
If `inactiveTokens` is `0` it would display `-` right? 

Maybe we should do a more robust check?

Reply:
Maybe it was intended to show `-` being 0 ? not sure it was like this

Reply:
I think that it was the intention because all the 0 are displayed like `-`